src/ontology/uberon-edit.obo

```diff
@@ -224439,6 +224439,18 @@ relationship: dc-contributor https://orcid.org/0000-0002-9185-3994
 relationship: part_of UBERON:0000101 ! lobe of lung
 property_value: dcterms-date "2023-07-20T12:43:29Z" xsd:dateTime
 
+[Term]
+id: UBERON:8600018
+name: neuroendocrine system
+def: "A regulatory system of the body that integrates the nervous system and the endocrine system. This system consists of a set of cells organized in single organs and diffuse elements, sharing co-production of amine hormone/transmitters, peptide hormone/transmitters and specific markers of neural determination." [PMID:15481802, Wikipedia:endocrinology]
```

Comment:
Consider using the term "neurohormone" as the current definition does not capture all neurohormones (e.g., steroid hormones).

Reply:
The second sentence: "involves" and "various organs" are vague.